test/Graphics/RenderWindow.test.cpp

```diff
@@ -1,8 +1,60 @@
+#include <SFML/Graphics/Image.hpp>
 #include <SFML/Graphics/RenderWindow.hpp>
```

Comment:
We need to ensure that `SFML/Graphics/RenderWindow.hpp` remains the first include. Including it first means we're testing that it includes all the headers required to build itself. See how I fixed this in [WindowBase.test.cpp](https://github.com/SFML/SFML/blob/master/test/Window/WindowBase.test.cpp).